packages/react/src/components/Breadcrumb/__snapshots__/Breadcrumb.test.tsx.snap

```diff
@@ -72,17 +72,19 @@ exports[`Breadcrumb rendering renders  correctly with custom overflow icon 1`] =
                       {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32380&commitId=f1bb87528f74eed032eb338b08049571d032595f&env=prod&clientType=FLUENTUI)


  <details><summary><b>(Dev-Only) Suggestions</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.chromium.png|4602| Changed|

</details>


  <details><summary><b>Button (compat)</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button (compat).Primary Anchor.chromium.png|283| Changed|

</details>


  <details><summary><b>Button Special Scenarios (compat)</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Special Scenarios (compat).no flex shrink.chromium.png|69| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>